code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/AuthIntegrationManager.java

```diff
@@ -0,0 +1,48 @@
+package io.cattle.platform.iaas.api.auth;
+
+import io.cattle.platform.iaas.api.auth.integration.interfaces.AuthConfig;
+import io.cattle.platform.iaas.api.auth.integration.interfaces.AuthConfigManager;
+import io.github.ibuildthecloud.gdapi.factory.SchemaFactory;
+import io.github.ibuildthecloud.gdapi.model.ListOptions;
+import io.github.ibuildthecloud.gdapi.request.resource.impl.AbstractNoOpResourceManager;
+
+import java.util.ArrayList;
+import java.util.Map;
+import javax.inject.Inject;
+
+public class AuthIntegrationManager extends AbstractNoOpResourceManager {
+
+    Map<String, AuthConfigManager> authConfigManagers;
```

Comment:
Nit: s there a reason this a map instead of list? Not a big, just noticed its only used in the class a basically a list.